File: app/src/main/java/com/kml/data/models/Work.java
package com.kml.data.models;

import com.google.gson.annotations.SerializedName;

public class Work
{
    @SerializedName("nazwaZadania")
    private String workName;

    @SerializedName("opisZadania")
    private String workDescription;

    @SerializedName("data")
    private String workDate;

    @SerializedName("czasWykonania")
    private String executionTime;

    public Work(String workName, String workDescription, String workDate, String executionTime)
    {
        this.workName = workName;
        this.workDescription = workDescription;
        this.workDate = workDate;
        this.executionTime = executionTime;
    }

    public String getWorkName()
    {
        return workName;
    }

    public String getWorkDescription()
    {
        return workDescription;
    }

    public String getWorkDate()
    {
        return workDate;
    }

    public String getExecutionTime()
    {
        return executionTime;
    }
}

File: app/src/main/java/com/kml/data/models/Work.kt
package com.kml.data.models

import com.google.gson.annotations.SerializedName

class Work(@field:SerializedName("nazwaZadania") val workName: String,
           @field:SerializedName("opisZadania") val workDescription: String,
           @field:SerializedName("data") val workDate: String,
           @field:SerializedName("czasWykonania") val executionTime: String)